[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248] mod_wsgi (pid=14965): Target WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi' cannot be loaded as Python module.
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248] mod_wsgi (pid=14965): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248] Traceback (most recent call last):
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/mixedmessage.wsgi", line 3, in <module>
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248]     from app import app as application
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/app/__init__.py", line 12, in <module>
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248]     from app import views, models
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/app/views.py", line 2, in <module>
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248]     from app import user_definitions, phrase_definitions, game_definitions
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 25
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248]     if finished=request.form['finished'] == '0':
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248]                ^
[Tue Mar 12 22:38:16 2013] [error] [client 171.66.133.248] SyntaxError: invalid syntax
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248] mod_wsgi (pid=14965): Target WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi' cannot be loaded as Python module.
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248] mod_wsgi (pid=14965): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248] Traceback (most recent call last):
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/mixedmessage.wsgi", line 3, in <module>
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248]     from app import app as application
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/app/__init__.py", line 12, in <module>
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248]     from app import views, models
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/app/views.py", line 2, in <module>
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248]     from app import user_definitions, phrase_definitions, game_definitions
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 25
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248]     if finished=request.form['finished'] == '0':
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248]                ^
[Tue Mar 12 22:39:27 2013] [error] [client 171.66.133.248] SyntaxError: invalid syntax
[Tue Mar 12 22:41:37 2013] [error] [client 171.66.133.248] mod_wsgi (pid=14965): Target WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi' cannot be loaded as Python module.
[Tue Mar 12 22:41:37 2013] [error] [client 171.66.133.248] mod_wsgi (pid=14965): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Tue Mar 12 22:41:37 2013] [error] [client 171.66.133.248] Traceback (most recent call last):
[Tue Mar 12 22:41:37 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/mixedmessage.wsgi", line 3, in <module>
[Tue Mar 12 22:41:37 2013] [error] [client 171.66.133.248]     from app import app as application
[Tue Mar 12 22:41:37 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/app/__init__.py", line 12, in <module>
[Tue Mar 12 22:41:37 2013] [error] [client 171.66.133.248]     from app import views, models
[Tue Mar 12 22:41:37 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/app/views.py", line 2, in <module>
[Tue Mar 12 22:41:37 2013] [error] [client 171.66.133.248]     from app import user_definitions, phrase_definitions, game_definitions
[Tue Mar 12 22:41:37 2013] [error] [client 171.66.133.248] ImportError: cannot import name user_definitions
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248] mod_wsgi (pid=15701): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248] Traceback (most recent call last):
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     return self.wsgi_app(environ, start_response)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     response = self.make_response(self.handle_exception(e))
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     response = self.full_dispatch_request()
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     rv = self.handle_user_exception(e)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     rv = self.dispatch_request()
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     return self.view_functions[rule.endpoint](**req.view_args)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/app/views.py", line 32, in create_game_result_view
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     return game_definitions.create_game_result()
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 54, in create_game_result
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     db.session.commit()
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/scoping.py", line 114, in do
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     return getattr(self.registry(), name)(*args, **kwargs)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 703, in commit
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     self.transaction.commit()
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 361, in commit
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     self._prepare_impl()
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 340, in _prepare_impl
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     self.session.flush()
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 1718, in flush
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     self._flush(objects)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 1789, in _flush
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     flush_context.execute()
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/unitofwork.py", line 331, in execute
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     rec.execute(self)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/unitofwork.py", line 475, in execute
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     uow
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/persistence.py", line 64, in save_obj
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     table, insert)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/persistence.py", line 558, in _emit_insert_statements
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     execute(statement, params)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1449, in execute
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     params)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1584, in _execute_clauseelement
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     compiled_sql, distilled_params
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1698, in _execute_context
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     context)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1691, in _execute_context
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     context)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/default.py", line 331, in do_execute
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]     cursor.execute(statement, parameters)
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248] IntegrityError: (IntegrityError) insert or update on table "result" violates foreign key constraint "result_user_id_fkey"
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248] DETAIL:  Key (user_id)=(0) is not present in table "user".
[Tue Mar 12 22:42:35 2013] [error] [client 171.66.133.248]  'INSERT INTO result (input_phrase, accuracy, wpm, duration, initiating, finished, created_at, user_id, game_id) VALUES (%(input_phrase)s, %(accuracy)s, %(wpm)s, %(duration)s, %(initiating)s, %(finished)s, %(created_at)s, %(user_id)s, %(game_id)s) RETURNING result.id' {'user_id': u'0', 'input_phrase': '', 'created_at': datetime.datetime(2013, 3, 12, 22, 42, 35, 630133), 'wpm': 0, 'finished': False, 'duration': 0, 'game_id': u'37', 'initiating': False, 'accuracy': 0}
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92] mod_wsgi (pid=8489): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]     return game_definitions.games_for_user()
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 76, in games_for_user
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92]     games = Game.query.filter_by(game_owner=user.id)
[Thu Mar 14 00:35:19 2013] [error] [client 171.66.132.92] AttributeError: 'NoneType' object has no attribute 'id'
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92] mod_wsgi (pid=8489): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]     return game_definitions.games_for_user()
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 76, in games_for_user
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92]     games = Game.query.filter_by(game_owner=user.id)
[Thu Mar 14 00:35:56 2013] [error] [client 171.66.132.92] AttributeError: 'NoneType' object has no attribute 'id'
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92] mod_wsgi (pid=12187): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 12, in create_user_view
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]     return user_definitions.create()
[Thu Mar 14 06:25:49 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/user_definitions.py", line 11, in create
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     new_user = create_user(user, email, password, True)
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/user_definitions.py", line 34, in create_user
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     db.session.commit()
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/scoping.py", line 114, in do
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     return getattr(self.registry(), name)(*args, **kwargs)
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 703, in commit
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     self.transaction.commit()
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 361, in commit
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     self._prepare_impl()
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 340, in _prepare_impl
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     self.session.flush()
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 1718, in flush
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     self._flush(objects)
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 1789, in _flush
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     flush_context.execute()
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/unitofwork.py", line 331, in execute
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     rec.execute(self)
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/unitofwork.py", line 475, in execute
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     uow
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/persistence.py", line 64, in save_obj
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     table, insert)
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/persistence.py", line 558, in _emit_insert_statements
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     execute(statement, params)
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1449, in execute
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     params)
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1584, in _execute_clauseelement
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     compiled_sql, distilled_params
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1698, in _execute_context
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     context)
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1691, in _execute_context
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     context)
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/default.py", line 331, in do_execute
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]     cursor.execute(statement, parameters)
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92] IntegrityError: (IntegrityError) duplicate key value violates unique constraint "ix_user_email"
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92] DETAIL:  Key (email)=(alexjaustin@gmail.com) already exists.
[Thu Mar 14 06:25:50 2013] [error] [client 171.66.132.92]  'INSERT INTO "user" (name, email, phone, password, subscribed, fb_userid, curr_coins, total_coins_earned, total_coins_purchased, games_played, friends_invited, created_at, recent_activity) VALUES (%(name)s, %(email)s, %(phone)s, %(password)s, %(subscribed)s, %(fb_userid)s, %(curr_coins)s, %(total_coins_earned)s, %(total_coins_purchased)s, %(games_played)s, %(friends_invited)s, %(created_at)s, %(recent_activity)s) RETURNING "user".id' {'friends_invited': None, 'fb_userid': None, 'name': u'Alex', 'subscribed': True, 'created_at': datetime.datetime(2013, 3, 14, 6, 25, 49, 736138), 'total_coins_purchased': None, 'total_coins_earned': None, 'phone': None, 'email': u'alexjaustin@gmail.com', 'games_played': None, 'recent_activity': None, 'password': u'aa120148', 'curr_coins': None}
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92] mod_wsgi (pid=12802): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 32, in create_game_result_view
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     return game_definitions.create_game_result()
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 59, in create_game_result
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     if not result.initiating:
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92] AttributeError: 'BaseQuery' object has no attribute 'initiating'
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92] mod_wsgi (pid=12802): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 32, in create_game_result_view
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     return game_definitions.create_game_result()
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 59, in create_game_result
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92]     if not result.initiating:
[Thu Mar 14 06:37:28 2013] [error] [client 171.66.132.92] AttributeError: 'BaseQuery' object has no attribute 'initiating'
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92] mod_wsgi (pid=23037): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 32, in create_game_result_view
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]     return game_definitions.create_game_result()
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 51, in create_game_result
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92]     increment_games_played(request.form['user_id']).first()
[Thu Mar 14 19:01:10 2013] [error] [client 171.66.132.92] AttributeError: 'NoneType' object has no attribute 'first'
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92] mod_wsgi (pid=24710): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     return game_definitions.games_for_user()
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 150, in games_for_user
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     resp = jsonify(data)
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/helpers.py", line 198, in jsonify
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     indent=None if request.is_xhr else 2), mimetype='application/json')
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/lib/python2.7/dist-packages/simplejson/__init__.py", line 296, in dumps
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     **kw).encode(obj)
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/lib/python2.7/dist-packages/simplejson/encoder.py", line 228, in encode
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     chunks = list(chunks)
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/lib/python2.7/dist-packages/simplejson/encoder.py", line 521, in _iterencode
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     for chunk in _iterencode_dict(o, _current_indent_level):
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/lib/python2.7/dist-packages/simplejson/encoder.py", line 487, in _iterencode_dict
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     for chunk in chunks:
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/lib/python2.7/dist-packages/simplejson/encoder.py", line 401, in _iterencode_list
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     for chunk in chunks:
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/lib/python2.7/dist-packages/simplejson/encoder.py", line 487, in _iterencode_dict
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     for chunk in chunks:
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/lib/python2.7/dist-packages/simplejson/encoder.py", line 531, in _iterencode
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     o = _default(o)
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]   File "/usr/lib/python2.7/dist-packages/simplejson/encoder.py", line 202, in default
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92]     raise TypeError(repr(o) + " is not JSON serializable")
[Thu Mar 14 20:47:53 2013] [error] [client 171.66.132.92] TypeError: datetime.datetime(2013, 3, 14, 19, 56, 37, 635444) is not JSON serializable
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92] mod_wsgi (pid=24874): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]     return game_definitions.games_for_user()
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 101, in games_for_user
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92]     'created_at' : game.created_at.to_s()
[Thu Mar 14 20:50:17 2013] [error] [client 171.66.132.92] UnboundLocalError: local variable 'game' referenced before assignment
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92] mod_wsgi (pid=25018): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]     return game_definitions.games_for_user()
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 101, in games_for_user
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92]     'created_at' : user.created_at.to_s()
[Thu Mar 14 20:51:55 2013] [error] [client 171.66.132.92] AttributeError: 'datetime.datetime' object has no attribute 'to_s'
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92] mod_wsgi (pid=25262): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]     return game_definitions.games_for_user()
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 113, in games_for_user
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92]     'responded_to' : game.responded_to.strftime('%m/%d/%Y'),
[Thu Mar 14 20:53:21 2013] [error] [client 171.66.132.92] AttributeError: 'NoneType' object has no attribute 'strftime'
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92] mod_wsgi (pid=25339): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]     return game_definitions.games_for_user()
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 87, in games_for_user
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92]     games = Game.query.filter_by(game_owner=user.id)
[Fri Mar 15 00:07:41 2013] [error] [client 171.66.132.92] AttributeError: 'NoneType' object has no attribute 'id'
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92] mod_wsgi (pid=25339): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]     return game_definitions.games_for_user()
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 87, in games_for_user
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92]     games = Game.query.filter_by(game_owner=user.id)
[Fri Mar 15 00:08:27 2013] [error] [client 171.66.132.92] AttributeError: 'NoneType' object has no attribute 'id'
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92] mod_wsgi (pid=25339): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]     return game_definitions.games_for_user()
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 87, in games_for_user
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92]     games = Game.query.filter_by(game_owner=user.id)
[Fri Mar 15 00:10:02 2013] [error] [client 171.66.132.92] AttributeError: 'NoneType' object has no attribute 'id'
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92] mod_wsgi (pid=25339): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]     return game_definitions.games_for_user()
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 87, in games_for_user
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92]     games = Game.query.filter_by(game_owner=user.id)
[Fri Mar 15 00:39:25 2013] [error] [client 171.66.132.92] AttributeError: 'NoneType' object has no attribute 'id'
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92] mod_wsgi (pid=30592): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92] Traceback (most recent call last):
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]     return self.wsgi_app(environ, start_response)
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]     response = self.make_response(self.handle_exception(e))
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]     response = self.full_dispatch_request()
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]     rv = self.handle_user_exception(e)
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]     rv = self.dispatch_request()
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]     return self.view_functions[rule.endpoint](**req.view_args)
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/views.py", line 32, in create_game_result_view
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]     return game_definitions.create_game_result()
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 63, in create_game_result
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92]     increment_games_played(request.form['user_id']).first()
[Fri Mar 15 04:03:20 2013] [error] [client 171.66.132.92] AttributeError: 'NoneType' object has no attribute 'first'
[Fri Mar 15 18:55:45 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: '250-8BITMIME\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Fri Mar 15 18:55:45 2013] [error] SIZE 35882577
[Fri Mar 15 18:55:45 2013] [error] 8BITMIME
[Fri Mar 15 18:55:45 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Fri Mar 15 18:55:45 2013] [error] ENHANCEDSTATUSCODES
[Fri Mar 15 18:55:45 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Fri Mar 15 18:55:45 2013] [error] send: u'mail FROM:<p@face.com> size=1153\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: '250 2.1.0 OK rl3sm10017625pbb.28 - gsmtp\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: retcode (250); Msg: 2.1.0 OK rl3sm10017625pbb.28 - gsmtp
[Fri Mar 15 18:55:45 2013] [error] send: u'rcpt TO:<joanne.yuan@gmail.com>\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: '250 2.1.5 OK rl3sm10017625pbb.28 - gsmtp\\r\\n'
[Fri Mar 15 18:55:45 2013] [error] reply: retcode (250); Msg: 2.1.5 OK rl3sm10017625pbb.28 - gsmtp
[Fri Mar 15 18:55:45 2013] [error] send: 'data\\r\\n'
[Fri Mar 15 18:55:46 2013] [error] reply: '354  Go ahead rl3sm10017625pbb.28 - gsmtp\\r\\n'
[Fri Mar 15 18:55:46 2013] [error] reply: retcode (354); Msg: Go ahead rl3sm10017625pbb.28 - gsmtp
[Fri Mar 15 18:55:46 2013] [error] data: (354, 'Go ahead rl3sm10017625pbb.28 - gsmtp')
[Fri Mar 15 18:55:46 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============6445049390617831635=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Poop Faceto a texting contest!\\r\\nFrom: MixedMessage challenges <p@face.com>\\r\\nTo: joanne.yuan@gmail.com\\r\\nDate: Fri, 15 Mar 2013 18:55:45 +0000\\r\\nMessage-ID: <20130315185544.15070.50985@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============6445049390617831635==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============2640247736503009826=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============2640247736503009826==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============2640247736503009826==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEpvYW5uZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3VyIGZy\\r\\naWVuZCBQb29wIEZhY2UgaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0ZXIh\\r\\nPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwvcD4K\\r\\nPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============2640247736503009826==--\\r\\n--===============6445049390617831635==--\\r\\n.\\r\\n'
[Fri Mar 15 18:55:47 2013] [error] reply: '250 2.0.0 OK 1363373788 rl3sm10017625pbb.28 - gsmtp\\r\\n'
[Fri Mar 15 18:55:47 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363373788 rl3sm10017625pbb.28 - gsmtp
[Fri Mar 15 18:55:47 2013] [error] data: (250, '2.0.0 OK 1363373788 rl3sm10017625pbb.28 - gsmtp')
[Fri Mar 15 18:55:47 2013] [error] send: 'quit\\r\\n'
[Fri Mar 15 18:55:47 2013] [error] reply: '221 2.0.0 closing connection rl3sm10017625pbb.28 - gsmtp\\r\\n'
[Fri Mar 15 18:55:47 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection rl3sm10017625pbb.28 - gsmtp
[Fri Mar 15 22:10:55 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: '250-8BITMIME\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Fri Mar 15 22:10:55 2013] [error] SIZE 35882577
[Fri Mar 15 22:10:55 2013] [error] 8BITMIME
[Fri Mar 15 22:10:55 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Fri Mar 15 22:10:55 2013] [error] ENHANCEDSTATUSCODES
[Fri Mar 15 22:10:55 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Fri Mar 15 22:10:55 2013] [error] send: u'mail FROM:<p@face.com> size=1140\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: '250 2.1.0 OK is1sm10635205pbc.15 - gsmtp\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: retcode (250); Msg: 2.1.0 OK is1sm10635205pbc.15 - gsmtp
[Fri Mar 15 22:10:55 2013] [error] send: u'rcpt TO:<joanne.yuan@gmail.com>\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: '250 2.1.5 OK is1sm10635205pbc.15 - gsmtp\\r\\n'
[Fri Mar 15 22:10:55 2013] [error] reply: retcode (250); Msg: 2.1.5 OK is1sm10635205pbc.15 - gsmtp
[Fri Mar 15 22:10:55 2013] [error] send: 'data\\r\\n'
[Fri Mar 15 22:10:56 2013] [error] reply: '354  Go ahead is1sm10635205pbc.15 - gsmtp\\r\\n'
[Fri Mar 15 22:10:56 2013] [error] reply: retcode (354); Msg: Go ahead is1sm10635205pbc.15 - gsmtp
[Fri Mar 15 22:10:56 2013] [error] data: (354, 'Go ahead is1sm10635205pbc.15 - gsmtp')
[Fri Mar 15 22:10:56 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============8724294720544654060=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Alexto a texting contest!\\r\\nFrom: MixedMessage challenges <p@face.com>\\r\\nTo: joanne.yuan@gmail.com\\r\\nDate: Fri, 15 Mar 2013 22:10:55 +0000\\r\\nMessage-ID: <20130315221054.15070.95940@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============8724294720544654060==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============2857625633800793866=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============2857625633800793866==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============2857625633800793866==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEpvYW5uZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3VyIGZy\\r\\naWVuZCBBbGV4IGluIGEgdGV4dCBvZmYhITwvcD4KPGgyPllvdSBjYW4gZG8gYmV0dGVyITwvYT48\\r\\nL2gyPgo8cD5Eb3dubG9hZCB0aGUgYXBwIGFuZCBiZWF0IHRoZW0gaG9ycmlibHk8L3A+CjxwPgpD\\r\\naGVlcnMsCjxicj4KQWxleAo8YnI+Ck1peGVkTWVzc2FnZSB0ZWFtCjwvcD4=\\r\\n\\r\\n--===============2857625633800793866==--\\r\\n--===============8724294720544654060==--\\r\\n.\\r\\n'
[Fri Mar 15 22:10:57 2013] [error] reply: '250 2.0.0 OK 1363385498 is1sm10635205pbc.15 - gsmtp\\r\\n'
[Fri Mar 15 22:10:57 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363385498 is1sm10635205pbc.15 - gsmtp
[Fri Mar 15 22:10:57 2013] [error] data: (250, '2.0.0 OK 1363385498 is1sm10635205pbc.15 - gsmtp')
[Fri Mar 15 22:10:57 2013] [error] send: 'quit\\r\\n'
[Fri Mar 15 22:10:57 2013] [error] reply: '221 2.0.0 closing connection is1sm10635205pbc.15 - gsmtp\\r\\n'
[Fri Mar 15 22:10:57 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection is1sm10635205pbc.15 - gsmtp
[Sat Mar 16 01:17:47 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sat Mar 16 01:17:47 2013] [error] SIZE 35882577
[Sat Mar 16 01:17:47 2013] [error] 8BITMIME
[Sat Mar 16 01:17:47 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sat Mar 16 01:17:47 2013] [error] ENHANCEDSTATUSCODES
[Sat Mar 16 01:17:47 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sat Mar 16 01:17:47 2013] [error] send: u'mail FROM:<alexjaustin@gmail.com> size=1151\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: '250 2.1.0 OK ri1sm11265707pbc.16 - gsmtp\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: retcode (250); Msg: 2.1.0 OK ri1sm11265707pbc.16 - gsmtp
[Sat Mar 16 01:17:47 2013] [error] send: u'rcpt TO:<joanne.yuan@gmail.com>\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: '250 2.1.5 OK ri1sm11265707pbc.16 - gsmtp\\r\\n'
[Sat Mar 16 01:17:47 2013] [error] reply: retcode (250); Msg: 2.1.5 OK ri1sm11265707pbc.16 - gsmtp
[Sat Mar 16 01:17:47 2013] [error] send: 'data\\r\\n'
[Sat Mar 16 01:17:48 2013] [error] reply: '354  Go ahead ri1sm11265707pbc.16 - gsmtp\\r\\n'
[Sat Mar 16 01:17:48 2013] [error] reply: retcode (354); Msg: Go ahead ri1sm11265707pbc.16 - gsmtp
[Sat Mar 16 01:17:48 2013] [error] data: (354, 'Go ahead ri1sm11265707pbc.16 - gsmtp')
[Sat Mar 16 01:17:48 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============1160596230314467640=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Alexto a texting contest!\\r\\nFrom: MixedMessage challenges <alexjaustin@gmail.com>\\r\\nTo: joanne.yuan@gmail.com\\r\\nDate: Sat, 16 Mar 2013 01:17:47 +0000\\r\\nMessage-ID: <20130316011746.15070.74836@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============1160596230314467640==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============4365997139356401982=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============4365997139356401982==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============4365997139356401982==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEpvYW5uZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3VyIGZy\\r\\naWVuZCBBbGV4IGluIGEgdGV4dCBvZmYhITwvcD4KPGgyPllvdSBjYW4gZG8gYmV0dGVyITwvYT48\\r\\nL2gyPgo8cD5Eb3dubG9hZCB0aGUgYXBwIGFuZCBiZWF0IHRoZW0gaG9ycmlibHk8L3A+CjxwPgpD\\r\\naGVlcnMsCjxicj4KQWxleAo8YnI+Ck1peGVkTWVzc2FnZSB0ZWFtCjwvcD4=\\r\\n\\r\\n--===============4365997139356401982==--\\r\\n--===============1160596230314467640==--\\r\\n.\\r\\n'
[Sat Mar 16 01:17:49 2013] [error] reply: '250 2.0.0 OK 1363396710 ri1sm11265707pbc.16 - gsmtp\\r\\n'
[Sat Mar 16 01:17:49 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363396710 ri1sm11265707pbc.16 - gsmtp
[Sat Mar 16 01:17:49 2013] [error] data: (250, '2.0.0 OK 1363396710 ri1sm11265707pbc.16 - gsmtp')
[Sat Mar 16 01:17:49 2013] [error] send: 'quit\\r\\n'
[Sat Mar 16 01:17:49 2013] [error] reply: '221 2.0.0 closing connection ri1sm11265707pbc.16 - gsmtp\\r\\n'
[Sat Mar 16 01:17:49 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection ri1sm11265707pbc.16 - gsmtp
[Sat Mar 16 01:20:35 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sat Mar 16 01:20:35 2013] [error] SIZE 35882577
[Sat Mar 16 01:20:35 2013] [error] 8BITMIME
[Sat Mar 16 01:20:35 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sat Mar 16 01:20:35 2013] [error] ENHANCEDSTATUSCODES
[Sat Mar 16 01:20:35 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sat Mar 16 01:20:35 2013] [error] send: u'mail FROM:<alexjaustin@gmail.com> size=1150\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: '250 2.1.0 OK cy4sm9746381pbc.13 - gsmtp\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: retcode (250); Msg: 2.1.0 OK cy4sm9746381pbc.13 - gsmtp
[Sat Mar 16 01:20:35 2013] [error] send: u'rcpt TO:<joanne.yuan@gmail.com>\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: '250 2.1.5 OK cy4sm9746381pbc.13 - gsmtp\\r\\n'
[Sat Mar 16 01:20:35 2013] [error] reply: retcode (250); Msg: 2.1.5 OK cy4sm9746381pbc.13 - gsmtp
[Sat Mar 16 01:20:35 2013] [error] send: 'data\\r\\n'
[Sat Mar 16 01:20:36 2013] [error] reply: '354  Go ahead cy4sm9746381pbc.13 - gsmtp\\r\\n'
[Sat Mar 16 01:20:36 2013] [error] reply: retcode (354); Msg: Go ahead cy4sm9746381pbc.13 - gsmtp
[Sat Mar 16 01:20:36 2013] [error] data: (354, 'Go ahead cy4sm9746381pbc.13 - gsmtp')
[Sat Mar 16 01:20:36 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============2124391593594228425=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Alexto a texting contest!\\r\\nFrom: MixedMessage challenges <alexjaustin@gmail.com>\\r\\nTo: joanne.yuan@gmail.com\\r\\nDate: Sat, 16 Mar 2013 01:20:35 +0000\\r\\nMessage-ID: <20130316012035.15070.2389@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============2124391593594228425==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============4871017376134415246=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============4871017376134415246==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============4871017376134415246==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEpvYW5uZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3VyIGZy\\r\\naWVuZCBBbGV4IGluIGEgdGV4dCBvZmYhITwvcD4KPGgyPllvdSBjYW4gZG8gYmV0dGVyITwvYT48\\r\\nL2gyPgo8cD5Eb3dubG9hZCB0aGUgYXBwIGFuZCBiZWF0IHRoZW0gaG9ycmlibHk8L3A+CjxwPgpD\\r\\naGVlcnMsCjxicj4KQWxleAo8YnI+Ck1peGVkTWVzc2FnZSB0ZWFtCjwvcD4=\\r\\n\\r\\n--===============4871017376134415246==--\\r\\n--===============2124391593594228425==--\\r\\n.\\r\\n'
[Sat Mar 16 01:20:36 2013] [error] reply: '250 2.0.0 OK 1363396878 cy4sm9746381pbc.13 - gsmtp\\r\\n'
[Sat Mar 16 01:20:36 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363396878 cy4sm9746381pbc.13 - gsmtp
[Sat Mar 16 01:20:36 2013] [error] data: (250, '2.0.0 OK 1363396878 cy4sm9746381pbc.13 - gsmtp')
[Sat Mar 16 01:20:36 2013] [error] send: 'quit\\r\\n'
[Sat Mar 16 01:20:36 2013] [error] reply: '221 2.0.0 closing connection cy4sm9746381pbc.13 - gsmtp\\r\\n'
[Sat Mar 16 01:20:36 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection cy4sm9746381pbc.13 - gsmtp
[Sat Mar 16 01:37:28 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sat Mar 16 01:37:28 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sat Mar 16 01:37:28 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sat Mar 16 01:37:28 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sat Mar 16 01:37:28 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sat Mar 16 01:37:28 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sat Mar 16 01:37:28 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sat Mar 16 01:37:28 2013] [error] SIZE 35882577
[Sat Mar 16 01:37:28 2013] [error] 8BITMIME
[Sat Mar 16 01:37:28 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sat Mar 16 01:37:28 2013] [error] ENHANCEDSTATUSCODES
[Sat Mar 16 01:37:28 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sat Mar 16 01:37:29 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sat Mar 16 01:37:29 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sat Mar 16 01:37:29 2013] [error] send: u'mail FROM:<alexjaustin@gmail.com> size=1144\\r\\n'
[Sat Mar 16 01:37:29 2013] [error] reply: '250 2.1.0 OK ol7sm11339250pbb.14 - gsmtp\\r\\n'
[Sat Mar 16 01:37:29 2013] [error] reply: retcode (250); Msg: 2.1.0 OK ol7sm11339250pbb.14 - gsmtp
[Sat Mar 16 01:37:29 2013] [error] send: u'rcpt TO:<p@face.com>\\r\\n'
[Sat Mar 16 01:37:29 2013] [error] reply: '250 2.1.5 OK ol7sm11339250pbb.14 - gsmtp\\r\\n'
[Sat Mar 16 01:37:29 2013] [error] reply: retcode (250); Msg: 2.1.5 OK ol7sm11339250pbb.14 - gsmtp
[Sat Mar 16 01:37:29 2013] [error] send: 'data\\r\\n'
[Sat Mar 16 01:37:29 2013] [error] reply: '354  Go ahead ol7sm11339250pbb.14 - gsmtp\\r\\n'
[Sat Mar 16 01:37:29 2013] [error] reply: retcode (354); Msg: Go ahead ol7sm11339250pbb.14 - gsmtp
[Sat Mar 16 01:37:29 2013] [error] data: (354, 'Go ahead ol7sm11339250pbb.14 - gsmtp')
[Sat Mar 16 01:37:29 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============2855620205502973633=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Alex to a texting contest!\\r\\nFrom: MixedMessage challenges <alexjaustin@gmail.com>\\r\\nTo: p@face.com\\r\\nDate: Sat, 16 Mar 2013 01:37:29 +0000\\r\\nMessage-ID: <20130316013728.21364.8813@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============2855620205502973633==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============6713375377387394344=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============6713375377387394344==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============6713375377387394344==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IFBvb3AgRmFjZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBBbGV4IGluIGEgdGV4dCBvZmYhITwvcD4KPGgyPllvdSBjYW4gZG8gYmV0dGVyITwv\\r\\nYT48L2gyPgo8cD5Eb3dubG9hZCB0aGUgYXBwIGFuZCBiZWF0IHRoZW0gaG9ycmlibHk8L3A+Cjxw\\r\\nPgpDaGVlcnMsCjxicj4KQWxleAo8YnI+Ck1peGVkTWVzc2FnZSB0ZWFtCjwvcD4=\\r\\n\\r\\n--===============6713375377387394344==--\\r\\n--===============2855620205502973633==--\\r\\n.\\r\\n'
[Sat Mar 16 01:37:30 2013] [error] reply: '250 2.0.0 OK 1363397891 ol7sm11339250pbb.14 - gsmtp\\r\\n'
[Sat Mar 16 01:37:30 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363397891 ol7sm11339250pbb.14 - gsmtp
[Sat Mar 16 01:37:30 2013] [error] data: (250, '2.0.0 OK 1363397891 ol7sm11339250pbb.14 - gsmtp')
[Sat Mar 16 01:37:30 2013] [error] send: 'quit\\r\\n'
[Sat Mar 16 01:37:30 2013] [error] reply: '221 2.0.0 closing connection ol7sm11339250pbb.14 - gsmtp\\r\\n'
[Sat Mar 16 01:37:30 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection ol7sm11339250pbb.14 - gsmtp
[Sat Mar 16 08:33:33 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sat Mar 16 08:33:33 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sat Mar 16 08:33:33 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sat Mar 16 08:33:33 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sat Mar 16 08:33:33 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sat Mar 16 08:33:33 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sat Mar 16 08:33:33 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sat Mar 16 08:33:33 2013] [error] SIZE 35882577
[Sat Mar 16 08:33:33 2013] [error] 8BITMIME
[Sat Mar 16 08:33:33 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sat Mar 16 08:33:33 2013] [error] ENHANCEDSTATUSCODES
[Sat Mar 16 08:33:33 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sat Mar 16 08:33:34 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sat Mar 16 08:33:34 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sat Mar 16 08:33:34 2013] [error] send: u'mail FROM:<p@face.com> size=1154\\r\\n'
[Sat Mar 16 08:33:34 2013] [error] reply: '250 2.1.0 OK ql7sm12890536pbb.2 - gsmtp\\r\\n'
[Sat Mar 16 08:33:34 2013] [error] reply: retcode (250); Msg: 2.1.0 OK ql7sm12890536pbb.2 - gsmtp
[Sat Mar 16 08:33:34 2013] [error] send: u'rcpt TO:<gbluth@gpails.com>\\r\\n'
[Sat Mar 16 08:33:34 2013] [error] reply: '250 2.1.5 OK ql7sm12890536pbb.2 - gsmtp\\r\\n'
[Sat Mar 16 08:33:34 2013] [error] reply: retcode (250); Msg: 2.1.5 OK ql7sm12890536pbb.2 - gsmtp
[Sat Mar 16 08:33:34 2013] [error] send: 'data\\r\\n'
[Sat Mar 16 08:33:34 2013] [error] reply: '354  Go ahead ql7sm12890536pbb.2 - gsmtp\\r\\n'
[Sat Mar 16 08:33:34 2013] [error] reply: retcode (354); Msg: Go ahead ql7sm12890536pbb.2 - gsmtp
[Sat Mar 16 08:33:34 2013] [error] data: (354, 'Go ahead ql7sm12890536pbb.2 - gsmtp')
[Sat Mar 16 08:33:34 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============2607913034064501036=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Poop Face to a texting contest!\\r\\nFrom: MixedMessage challenges <p@face.com>\\r\\nTo: gbluth@gpails.com\\r\\nDate: Sat, 16 Mar 2013 08:33:34 +0000\\r\\nMessage-ID: <20130316083333.21364.43840@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============2607913034064501036==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============2610702863940304793=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============2610702863940304793==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============2610702863940304793==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEdvYiBCbHV0aCw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBQb29wIEZhY2UgaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0\\r\\nZXIhPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwv\\r\\ncD4KPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============2610702863940304793==--\\r\\n--===============2607913034064501036==--\\r\\n.\\r\\n'
[Sat Mar 16 08:33:35 2013] [error] reply: '250 2.0.0 OK 1363422857 ql7sm12890536pbb.2 - gsmtp\\r\\n'
[Sat Mar 16 08:33:35 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363422857 ql7sm12890536pbb.2 - gsmtp
[Sat Mar 16 08:33:35 2013] [error] data: (250, '2.0.0 OK 1363422857 ql7sm12890536pbb.2 - gsmtp')
[Sat Mar 16 08:33:35 2013] [error] send: 'quit\\r\\n'
[Sat Mar 16 08:33:35 2013] [error] reply: '221 2.0.0 closing connection ql7sm12890536pbb.2 - gsmtp\\r\\n'
[Sat Mar 16 08:33:35 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection ql7sm12890536pbb.2 - gsmtp
[Sat Mar 16 18:27:50 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sat Mar 16 18:27:50 2013] [error] SIZE 35882577
[Sat Mar 16 18:27:50 2013] [error] 8BITMIME
[Sat Mar 16 18:27:50 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sat Mar 16 18:27:50 2013] [error] ENHANCEDSTATUSCODES
[Sat Mar 16 18:27:50 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sat Mar 16 18:27:50 2013] [error] send: u'mail FROM:<p@face.com> size=1154\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: '250 2.1.0 OK 4sm14237723pbn.23 - gsmtp\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: retcode (250); Msg: 2.1.0 OK 4sm14237723pbn.23 - gsmtp
[Sat Mar 16 18:27:50 2013] [error] send: u'rcpt TO:<joanne.yuan@gmail.com>\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: '250 2.1.5 OK 4sm14237723pbn.23 - gsmtp\\r\\n'
[Sat Mar 16 18:27:50 2013] [error] reply: retcode (250); Msg: 2.1.5 OK 4sm14237723pbn.23 - gsmtp
[Sat Mar 16 18:27:50 2013] [error] send: 'data\\r\\n'
[Sat Mar 16 18:27:51 2013] [error] reply: '354  Go ahead 4sm14237723pbn.23 - gsmtp\\r\\n'
[Sat Mar 16 18:27:51 2013] [error] reply: retcode (354); Msg: Go ahead 4sm14237723pbn.23 - gsmtp
[Sat Mar 16 18:27:51 2013] [error] data: (354, 'Go ahead 4sm14237723pbn.23 - gsmtp')
[Sat Mar 16 18:27:51 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============5562120889184418468=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Poop Face to a texting contest!\\r\\nFrom: MixedMessage challenges <p@face.com>\\r\\nTo: joanne.yuan@gmail.com\\r\\nDate: Sat, 16 Mar 2013 18:27:50 +0000\\r\\nMessage-ID: <20130316182750.21364.34494@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============5562120889184418468==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============3451076693072565590=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============3451076693072565590==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============3451076693072565590==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEpvYW5uZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3VyIGZy\\r\\naWVuZCBQb29wIEZhY2UgaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0ZXIh\\r\\nPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwvcD4K\\r\\nPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============3451076693072565590==--\\r\\n--===============5562120889184418468==--\\r\\n.\\r\\n'
[Sat Mar 16 18:27:52 2013] [error] reply: '250 2.0.0 OK 1363458515 4sm14237723pbn.23 - gsmtp\\r\\n'
[Sat Mar 16 18:27:52 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363458515 4sm14237723pbn.23 - gsmtp
[Sat Mar 16 18:27:52 2013] [error] data: (250, '2.0.0 OK 1363458515 4sm14237723pbn.23 - gsmtp')
[Sat Mar 16 18:27:52 2013] [error] send: 'quit\\r\\n'
[Sat Mar 16 18:27:52 2013] [error] reply: '221 2.0.0 closing connection 4sm14237723pbn.23 - gsmtp\\r\\n'
[Sat Mar 16 18:27:52 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection 4sm14237723pbn.23 - gsmtp
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110] mod_wsgi (pid=21364): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110] Traceback (most recent call last):
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]     return self.wsgi_app(environ, start_response)
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]     response = self.make_response(self.handle_exception(e))
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]     response = self.full_dispatch_request()
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]     rv = self.handle_user_exception(e)
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]     rv = self.dispatch_request()
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]     return self.view_functions[rule.endpoint](**req.view_args)
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]     return game_definitions.games_for_user()
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 96, in games_for_user
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110]     gameResults = Result.query.filter_by(user_id=user.id)
[Sat Mar 16 18:31:35 2013] [error] [client 171.66.130.110] AttributeError: 'NoneType' object has no attribute 'id'
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110] mod_wsgi (pid=21364): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110] Traceback (most recent call last):
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]     return self.wsgi_app(environ, start_response)
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]     response = self.make_response(self.handle_exception(e))
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]     response = self.full_dispatch_request()
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]     rv = self.handle_user_exception(e)
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]     rv = self.dispatch_request()
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]     return self.view_functions[rule.endpoint](**req.view_args)
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]     return game_definitions.games_for_user()
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 96, in games_for_user
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110]     gameResults = Result.query.filter_by(user_id=user.id)
[Sat Mar 16 18:32:20 2013] [error] [client 171.66.130.110] AttributeError: 'NoneType' object has no attribute 'id'
[Sat Mar 16 18:34:08 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sat Mar 16 18:34:08 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sat Mar 16 18:34:08 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sat Mar 16 18:34:08 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sat Mar 16 18:34:08 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sat Mar 16 18:34:08 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sat Mar 16 18:34:08 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sat Mar 16 18:34:08 2013] [error] SIZE 35882577
[Sat Mar 16 18:34:08 2013] [error] 8BITMIME
[Sat Mar 16 18:34:08 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sat Mar 16 18:34:08 2013] [error] ENHANCEDSTATUSCODES
[Sat Mar 16 18:34:08 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sat Mar 16 18:34:09 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sat Mar 16 18:34:09 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sat Mar 16 18:34:09 2013] [error] send: u'mail FROM:<p@face.com> size=1147\\r\\n'
[Sat Mar 16 18:34:09 2013] [error] reply: '250 2.1.0 OK wi7sm1736993pac.9 - gsmtp\\r\\n'
[Sat Mar 16 18:34:09 2013] [error] reply: retcode (250); Msg: 2.1.0 OK wi7sm1736993pac.9 - gsmtp
[Sat Mar 16 18:34:09 2013] [error] send: u'rcpt TO:<gbluth@gpails.com>\\r\\n'
[Sat Mar 16 18:34:09 2013] [error] reply: '250 2.1.5 OK wi7sm1736993pac.9 - gsmtp\\r\\n'
[Sat Mar 16 18:34:09 2013] [error] reply: retcode (250); Msg: 2.1.5 OK wi7sm1736993pac.9 - gsmtp
[Sat Mar 16 18:34:09 2013] [error] send: 'data\\r\\n'
[Sat Mar 16 18:34:09 2013] [error] reply: '354  Go ahead wi7sm1736993pac.9 - gsmtp\\r\\n'
[Sat Mar 16 18:34:09 2013] [error] reply: retcode (354); Msg: Go ahead wi7sm1736993pac.9 - gsmtp
[Sat Mar 16 18:34:09 2013] [error] data: (354, 'Go ahead wi7sm1736993pac.9 - gsmtp')
[Sat Mar 16 18:34:09 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============7962014949464454441=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by P Face to a texting contest!\\r\\nFrom: MixedMessage challenges <p@face.com>\\r\\nTo: gbluth@gpails.com\\r\\nDate: Sat, 16 Mar 2013 18:34:09 +0000\\r\\nMessage-ID: <20130316183408.21364.79055@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============7962014949464454441==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============2053466862885275283=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============2053466862885275283==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============2053466862885275283==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEdvYiBCbHV0aCw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBQIEZhY2UgaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0ZXIh\\r\\nPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwvcD4K\\r\\nPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============2053466862885275283==--\\r\\n--===============7962014949464454441==--\\r\\n.\\r\\n'
[Sat Mar 16 18:34:11 2013] [error] reply: '250 2.0.0 OK 1363458894 wi7sm1736993pac.9 - gsmtp\\r\\n'
[Sat Mar 16 18:34:11 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363458894 wi7sm1736993pac.9 - gsmtp
[Sat Mar 16 18:34:11 2013] [error] data: (250, '2.0.0 OK 1363458894 wi7sm1736993pac.9 - gsmtp')
[Sat Mar 16 18:34:11 2013] [error] send: 'quit\\r\\n'
[Sat Mar 16 18:34:11 2013] [error] reply: '221 2.0.0 closing connection wi7sm1736993pac.9 - gsmtp\\r\\n'
[Sat Mar 16 18:34:11 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection wi7sm1736993pac.9 - gsmtp
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110] mod_wsgi (pid=21364): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110] Traceback (most recent call last):
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]     return self.wsgi_app(environ, start_response)
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]     response = self.make_response(self.handle_exception(e))
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]     response = self.full_dispatch_request()
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]     rv = self.handle_user_exception(e)
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]     rv = self.dispatch_request()
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]     return self.view_functions[rule.endpoint](**req.view_args)
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/views.py", line 36, in grab_games_for_user
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]     return game_definitions.games_for_user()
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/game_definitions.py", line 96, in games_for_user
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110]     gameResults = Result.query.filter_by(user_id=user.id)
[Sat Mar 16 18:35:27 2013] [error] [client 171.66.130.110] AttributeError: 'NoneType' object has no attribute 'id'
[Sat Mar 16 18:53:32 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sat Mar 16 18:53:32 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sat Mar 16 18:53:32 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sat Mar 16 18:53:32 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sat Mar 16 18:53:32 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sat Mar 16 18:53:32 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sat Mar 16 18:53:32 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sat Mar 16 18:53:32 2013] [error] SIZE 35882577
[Sat Mar 16 18:53:32 2013] [error] 8BITMIME
[Sat Mar 16 18:53:32 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sat Mar 16 18:53:32 2013] [error] ENHANCEDSTATUSCODES
[Sat Mar 16 18:53:32 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sat Mar 16 18:53:33 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sat Mar 16 18:53:33 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sat Mar 16 18:53:33 2013] [error] send: u'mail FROM:<p@face.com> size=1146\\r\\n'
[Sat Mar 16 18:53:33 2013] [error] reply: '250 2.1.0 OK xz5sm14289851pbb.25 - gsmtp\\r\\n'
[Sat Mar 16 18:53:33 2013] [error] reply: retcode (250); Msg: 2.1.0 OK xz5sm14289851pbb.25 - gsmtp
[Sat Mar 16 18:53:33 2013] [error] send: u'rcpt TO:<gbluth@gpails.com>\\r\\n'
[Sat Mar 16 18:53:33 2013] [error] reply: '250 2.1.5 OK xz5sm14289851pbb.25 - gsmtp\\r\\n'
[Sat Mar 16 18:53:33 2013] [error] reply: retcode (250); Msg: 2.1.5 OK xz5sm14289851pbb.25 - gsmtp
[Sat Mar 16 18:53:33 2013] [error] send: 'data\\r\\n'
[Sat Mar 16 18:53:33 2013] [error] reply: '354  Go ahead xz5sm14289851pbb.25 - gsmtp\\r\\n'
[Sat Mar 16 18:53:33 2013] [error] reply: retcode (354); Msg: Go ahead xz5sm14289851pbb.25 - gsmtp
[Sat Mar 16 18:53:33 2013] [error] data: (354, 'Go ahead xz5sm14289851pbb.25 - gsmtp')
[Sat Mar 16 18:53:33 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============8981852507630866982=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by P face to a texting contest!\\r\\nFrom: MixedMessage challenges <p@face.com>\\r\\nTo: gbluth@gpails.com\\r\\nDate: Sat, 16 Mar 2013 18:53:33 +0000\\r\\nMessage-ID: <20130316185332.21364.5851@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============8981852507630866982==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============7335640631282933585=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============7335640631282933585==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============7335640631282933585==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEdvYiBCbHV0aCw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBQIGZhY2UgaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0ZXIh\\r\\nPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwvcD4K\\r\\nPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============7335640631282933585==--\\r\\n--===============8981852507630866982==--\\r\\n.\\r\\n'
[Sat Mar 16 18:53:34 2013] [error] reply: '250 2.0.0 OK 1363460057 xz5sm14289851pbb.25 - gsmtp\\r\\n'
[Sat Mar 16 18:53:34 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363460057 xz5sm14289851pbb.25 - gsmtp
[Sat Mar 16 18:53:34 2013] [error] data: (250, '2.0.0 OK 1363460057 xz5sm14289851pbb.25 - gsmtp')
[Sat Mar 16 18:53:34 2013] [error] send: 'quit\\r\\n'
[Sat Mar 16 18:53:34 2013] [error] reply: '221 2.0.0 closing connection xz5sm14289851pbb.25 - gsmtp\\r\\n'
[Sat Mar 16 18:53:34 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection xz5sm14289851pbb.25 - gsmtp
[Sat Mar 16 18:57:28 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sat Mar 16 18:57:28 2013] [error] SIZE 35882577
[Sat Mar 16 18:57:28 2013] [error] 8BITMIME
[Sat Mar 16 18:57:28 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sat Mar 16 18:57:28 2013] [error] ENHANCEDSTATUSCODES
[Sat Mar 16 18:57:28 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sat Mar 16 18:57:28 2013] [error] send: u'mail FROM:<p@face.com> size=1147\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: '250 2.1.0 OK lb8sm1806386pab.13 - gsmtp\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: retcode (250); Msg: 2.1.0 OK lb8sm1806386pab.13 - gsmtp
[Sat Mar 16 18:57:28 2013] [error] send: u'rcpt TO:<gbluth@gpails.com>\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: '250 2.1.5 OK lb8sm1806386pab.13 - gsmtp\\r\\n'
[Sat Mar 16 18:57:28 2013] [error] reply: retcode (250); Msg: 2.1.5 OK lb8sm1806386pab.13 - gsmtp
[Sat Mar 16 18:57:28 2013] [error] send: 'data\\r\\n'
[Sat Mar 16 18:57:29 2013] [error] reply: '354  Go ahead lb8sm1806386pab.13 - gsmtp\\r\\n'
[Sat Mar 16 18:57:29 2013] [error] reply: retcode (354); Msg: Go ahead lb8sm1806386pab.13 - gsmtp
[Sat Mar 16 18:57:29 2013] [error] data: (354, 'Go ahead lb8sm1806386pab.13 - gsmtp')
[Sat Mar 16 18:57:29 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============0442998153715019894=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by P Face to a texting contest!\\r\\nFrom: MixedMessage challenges <p@face.com>\\r\\nTo: gbluth@gpails.com\\r\\nDate: Sat, 16 Mar 2013 18:57:28 +0000\\r\\nMessage-ID: <20130316185728.21364.38479@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============0442998153715019894==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============2817107299351011351=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============2817107299351011351==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============2817107299351011351==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEdvYiBCbHV0aCw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBQIEZhY2UgaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0ZXIh\\r\\nPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwvcD4K\\r\\nPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============2817107299351011351==--\\r\\n--===============0442998153715019894==--\\r\\n.\\r\\n'
[Sat Mar 16 18:57:29 2013] [error] reply: '250 2.0.0 OK 1363460292 lb8sm1806386pab.13 - gsmtp\\r\\n'
[Sat Mar 16 18:57:29 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363460292 lb8sm1806386pab.13 - gsmtp
[Sat Mar 16 18:57:29 2013] [error] data: (250, '2.0.0 OK 1363460292 lb8sm1806386pab.13 - gsmtp')
[Sat Mar 16 18:57:29 2013] [error] send: 'quit\\r\\n'
[Sat Mar 16 18:57:29 2013] [error] reply: '221 2.0.0 closing connection lb8sm1806386pab.13 - gsmtp\\r\\n'
[Sat Mar 16 18:57:29 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection lb8sm1806386pab.13 - gsmtp
[Sun Mar 17 00:18:43 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 00:18:43 2013] [error] SIZE 35882577
[Sun Mar 17 00:18:43 2013] [error] 8BITMIME
[Sun Mar 17 00:18:43 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 00:18:43 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 00:18:43 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 00:18:43 2013] [error] send: u'mail FROM:<p@face.com> size=1141\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: '250 2.1.0 OK qd8sm14886616pbc.29 - gsmtp\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: retcode (250); Msg: 2.1.0 OK qd8sm14886616pbc.29 - gsmtp
[Sun Mar 17 00:18:43 2013] [error] send: u'rcpt TO:<gbluth@gpails.com>\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: '250 2.1.5 OK qd8sm14886616pbc.29 - gsmtp\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: retcode (250); Msg: 2.1.5 OK qd8sm14886616pbc.29 - gsmtp
[Sun Mar 17 00:18:43 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: '354  Go ahead qd8sm14886616pbc.29 - gsmtp\\r\\n'
[Sun Mar 17 00:18:43 2013] [error] reply: retcode (354); Msg: Go ahead qd8sm14886616pbc.29 - gsmtp
[Sun Mar 17 00:18:43 2013] [error] data: (354, 'Go ahead qd8sm14886616pbc.29 - gsmtp')
[Sun Mar 17 00:18:43 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============8892441431732574044=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Alex to a texting contest!\\r\\nFrom: MixedMessage challenges <p@face.com>\\r\\nTo: gbluth@gpails.com\\r\\nDate: Sun, 17 Mar 2013 00:18:43 +0000\\r\\nMessage-ID: <20130317001842.21364.27844@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============8892441431732574044==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============2972697579045571130=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============2972697579045571130==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============2972697579045571130==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEdvYiBCbHV0aCw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBBbGV4IGluIGEgdGV4dCBvZmYhITwvcD4KPGgyPllvdSBjYW4gZG8gYmV0dGVyITwv\\r\\nYT48L2gyPgo8cD5Eb3dubG9hZCB0aGUgYXBwIGFuZCBiZWF0IHRoZW0gaG9ycmlibHk8L3A+Cjxw\\r\\nPgpDaGVlcnMsCjxicj4KQWxleAo8YnI+Ck1peGVkTWVzc2FnZSB0ZWFtCjwvcD4=\\r\\n\\r\\n--===============2972697579045571130==--\\r\\n--===============8892441431732574044==--\\r\\n.\\r\\n'
[Sun Mar 17 00:18:44 2013] [error] reply: '250 2.0.0 OK 1363479568 qd8sm14886616pbc.29 - gsmtp\\r\\n'
[Sun Mar 17 00:18:44 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363479568 qd8sm14886616pbc.29 - gsmtp
[Sun Mar 17 00:18:44 2013] [error] data: (250, '2.0.0 OK 1363479568 qd8sm14886616pbc.29 - gsmtp')
[Sun Mar 17 00:18:44 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 00:18:44 2013] [error] reply: '221 2.0.0 closing connection qd8sm14886616pbc.29 - gsmtp\\r\\n'
[Sun Mar 17 00:18:44 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection qd8sm14886616pbc.29 - gsmtp
[Sun Mar 17 00:59:29 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 00:59:29 2013] [error] SIZE 35882577
[Sun Mar 17 00:59:29 2013] [error] 8BITMIME
[Sun Mar 17 00:59:29 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 00:59:29 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 00:59:29 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 00:59:29 2013] [error] send: u'mail FROM:<p@face.com> size=1147\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: '250 2.1.0 OK qb10sm14976941pbb.43 - gsmtp\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: retcode (250); Msg: 2.1.0 OK qb10sm14976941pbb.43 - gsmtp
[Sun Mar 17 00:59:29 2013] [error] send: u'rcpt TO:<joanne.yuan@gmail.com>\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: '250 2.1.5 OK qb10sm14976941pbb.43 - gsmtp\\r\\n'
[Sun Mar 17 00:59:29 2013] [error] reply: retcode (250); Msg: 2.1.5 OK qb10sm14976941pbb.43 - gsmtp
[Sun Mar 17 00:59:29 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 00:59:30 2013] [error] reply: '354  Go ahead qb10sm14976941pbb.43 - gsmtp\\r\\n'
[Sun Mar 17 00:59:30 2013] [error] reply: retcode (354); Msg: Go ahead qb10sm14976941pbb.43 - gsmtp
[Sun Mar 17 00:59:30 2013] [error] data: (354, 'Go ahead qb10sm14976941pbb.43 - gsmtp')
[Sun Mar 17 00:59:30 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============0247619000374669193=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by P Face to a texting contest!\\r\\nFrom: MixedMessage challenges <p@face.com>\\r\\nTo: joanne.yuan@gmail.com\\r\\nDate: Sun, 17 Mar 2013 00:59:29 +0000\\r\\nMessage-ID: <20130317005928.21364.66259@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============0247619000374669193==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============2856119117662992937=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============2856119117662992937==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============2856119117662992937==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEpvYW5uZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3VyIGZy\\r\\naWVuZCBQIEZhY2UgaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0ZXIhPC9h\\r\\nPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwvcD4KPHA+\\r\\nCkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============2856119117662992937==--\\r\\n--===============0247619000374669193==--\\r\\n.\\r\\n'
[Sun Mar 17 00:59:30 2013] [error] reply: '250 2.0.0 OK 1363482014 qb10sm14976941pbb.43 - gsmtp\\r\\n'
[Sun Mar 17 00:59:30 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363482014 qb10sm14976941pbb.43 - gsmtp
[Sun Mar 17 00:59:30 2013] [error] data: (250, '2.0.0 OK 1363482014 qb10sm14976941pbb.43 - gsmtp')
[Sun Mar 17 00:59:30 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 00:59:30 2013] [error] reply: '221 2.0.0 closing connection qb10sm14976941pbb.43 - gsmtp\\r\\n'
[Sun Mar 17 00:59:30 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection qb10sm14976941pbb.43 - gsmtp
[Sun Mar 17 02:19:07 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 02:19:07 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 02:19:07 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 02:19:07 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 02:19:07 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 02:19:07 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 02:19:07 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 02:19:07 2013] [error] SIZE 35882577
[Sun Mar 17 02:19:07 2013] [error] 8BITMIME
[Sun Mar 17 02:19:07 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 02:19:07 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 02:19:07 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 02:19:07 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 02:19:07 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 02:19:07 2013] [error] send: u'mail FROM:<jorge@gpails.com> size=1158\\r\\n'
[Sun Mar 17 02:19:07 2013] [error] reply: '250 2.1.0 OK lb8sm2765281pab.13 - gsmtp\\r\\n'
[Sun Mar 17 02:19:07 2013] [error] reply: retcode (250); Msg: 2.1.0 OK lb8sm2765281pab.13 - gsmtp
[Sun Mar 17 02:19:07 2013] [error] send: u'rcpt TO:<bbaggins@gpails.com>\\r\\n'
[Sun Mar 17 02:19:08 2013] [error] reply: '250 2.1.5 OK lb8sm2765281pab.13 - gsmtp\\r\\n'
[Sun Mar 17 02:19:08 2013] [error] reply: retcode (250); Msg: 2.1.5 OK lb8sm2765281pab.13 - gsmtp
[Sun Mar 17 02:19:08 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 02:19:08 2013] [error] reply: '354  Go ahead lb8sm2765281pab.13 - gsmtp\\r\\n'
[Sun Mar 17 02:19:08 2013] [error] reply: retcode (354); Msg: Go ahead lb8sm2765281pab.13 - gsmtp
[Sun Mar 17 02:19:08 2013] [error] data: (354, 'Go ahead lb8sm2765281pab.13 - gsmtp')
[Sun Mar 17 02:19:08 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============4229873072000445417=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Jorge to a texting contest!\\r\\nFrom: MixedMessage challenges <jorge@gpails.com>\\r\\nTo: bbaggins@gpails.com\\r\\nDate: Sun, 17 Mar 2013 02:19:07 +0000\\r\\nMessage-ID: <20130317021907.21364.19893@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============4229873072000445417==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============3333300171558299040=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============3333300171558299040==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============3333300171558299040==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEJpbGJvIEJhZ2dpbnMsPC9wPgo8cD5Zb3UgaGF2ZSBiZWVuIGNoYWxsZW5nZWQgYnkg\\r\\neW91ciBmcmllbmQgSm9yZ2UgaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0\\r\\nZXIhPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwv\\r\\ncD4KPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============3333300171558299040==--\\r\\n--===============4229873072000445417==--\\r\\n.\\r\\n'
[Sun Mar 17 02:19:09 2013] [error] reply: '250 2.0.0 OK 1363486793 lb8sm2765281pab.13 - gsmtp\\r\\n'
[Sun Mar 17 02:19:09 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363486793 lb8sm2765281pab.13 - gsmtp
[Sun Mar 17 02:19:09 2013] [error] data: (250, '2.0.0 OK 1363486793 lb8sm2765281pab.13 - gsmtp')
[Sun Mar 17 02:19:09 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 02:19:09 2013] [error] reply: '221 2.0.0 closing connection lb8sm2765281pab.13 - gsmtp\\r\\n'
[Sun Mar 17 02:19:09 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection lb8sm2765281pab.13 - gsmtp
[Sun Mar 17 02:22:37 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 02:22:37 2013] [error] SIZE 35882577
[Sun Mar 17 02:22:37 2013] [error] 8BITMIME
[Sun Mar 17 02:22:37 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 02:22:37 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 02:22:37 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 02:22:37 2013] [error] send: u'mail FROM:<mbongos@gpails.com> size=1160\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: '250 2.1.0 OK gn2sm15175102pbb.22 - gsmtp\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: retcode (250); Msg: 2.1.0 OK gn2sm15175102pbb.22 - gsmtp
[Sun Mar 17 02:22:37 2013] [error] send: u'rcpt TO:<bbaggins@gpails.com>\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: '250 2.1.5 OK gn2sm15175102pbb.22 - gsmtp\\r\\n'
[Sun Mar 17 02:22:37 2013] [error] reply: retcode (250); Msg: 2.1.5 OK gn2sm15175102pbb.22 - gsmtp
[Sun Mar 17 02:22:37 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 02:22:38 2013] [error] reply: '354  Go ahead gn2sm15175102pbb.22 - gsmtp\\r\\n'
[Sun Mar 17 02:22:38 2013] [error] reply: retcode (354); Msg: Go ahead gn2sm15175102pbb.22 - gsmtp
[Sun Mar 17 02:22:38 2013] [error] data: (354, 'Go ahead gn2sm15175102pbb.22 - gsmtp')
[Sun Mar 17 02:22:38 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============4340526486489659381=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Mongo to a texting contest!\\r\\nFrom: MixedMessage challenges <mbongos@gpails.com>\\r\\nTo: bbaggins@gpails.com\\r\\nDate: Sun, 17 Mar 2013 02:22:37 +0000\\r\\nMessage-ID: <20130317022237.21364.29363@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============4340526486489659381==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============8075036831367822548=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============8075036831367822548==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============8075036831367822548==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEJpbGJvIEJhZ2dpbnMsPC9wPgo8cD5Zb3UgaGF2ZSBiZWVuIGNoYWxsZW5nZWQgYnkg\\r\\neW91ciBmcmllbmQgTW9uZ28gaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0\\r\\nZXIhPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwv\\r\\ncD4KPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============8075036831367822548==--\\r\\n--===============4340526486489659381==--\\r\\n.\\r\\n'
[Sun Mar 17 02:22:38 2013] [error] reply: '250 2.0.0 OK 1363487002 gn2sm15175102pbb.22 - gsmtp\\r\\n'
[Sun Mar 17 02:22:38 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363487002 gn2sm15175102pbb.22 - gsmtp
[Sun Mar 17 02:22:38 2013] [error] data: (250, '2.0.0 OK 1363487002 gn2sm15175102pbb.22 - gsmtp')
[Sun Mar 17 02:22:38 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 02:22:38 2013] [error] reply: '221 2.0.0 closing connection gn2sm15175102pbb.22 - gsmtp\\r\\n'
[Sun Mar 17 02:22:38 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection gn2sm15175102pbb.22 - gsmtp
[Sun Mar 17 02:24:51 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 02:24:51 2013] [error] SIZE 35882577
[Sun Mar 17 02:24:51 2013] [error] 8BITMIME
[Sun Mar 17 02:24:51 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 02:24:51 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 02:24:51 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 02:24:51 2013] [error] send: u'mail FROM:<mbongos@gpails.com> size=1160\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: '250 2.1.0 OK gn2sm15177636pbb.22 - gsmtp\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: retcode (250); Msg: 2.1.0 OK gn2sm15177636pbb.22 - gsmtp
[Sun Mar 17 02:24:51 2013] [error] send: u'rcpt TO:<bbaggins@gpails.com>\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: '250 2.1.5 OK gn2sm15177636pbb.22 - gsmtp\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: retcode (250); Msg: 2.1.5 OK gn2sm15177636pbb.22 - gsmtp
[Sun Mar 17 02:24:51 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: '354  Go ahead gn2sm15177636pbb.22 - gsmtp\\r\\n'
[Sun Mar 17 02:24:51 2013] [error] reply: retcode (354); Msg: Go ahead gn2sm15177636pbb.22 - gsmtp
[Sun Mar 17 02:24:51 2013] [error] data: (354, 'Go ahead gn2sm15177636pbb.22 - gsmtp')
[Sun Mar 17 02:24:51 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============5100251953853153429=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Mongo to a texting contest!\\r\\nFrom: MixedMessage challenges <mbongos@gpails.com>\\r\\nTo: bbaggins@gpails.com\\r\\nDate: Sun, 17 Mar 2013 02:24:51 +0000\\r\\nMessage-ID: <20130317022450.21364.85579@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============5100251953853153429==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============8471855098144151455=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============8471855098144151455==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============8471855098144151455==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEJpbGJvIEJhZ2dpbnMsPC9wPgo8cD5Zb3UgaGF2ZSBiZWVuIGNoYWxsZW5nZWQgYnkg\\r\\neW91ciBmcmllbmQgTW9uZ28gaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0\\r\\nZXIhPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwv\\r\\ncD4KPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============8471855098144151455==--\\r\\n--===============5100251953853153429==--\\r\\n.\\r\\n'
[Sun Mar 17 02:24:52 2013] [error] reply: '250 2.0.0 OK 1363487135 gn2sm15177636pbb.22 - gsmtp\\r\\n'
[Sun Mar 17 02:24:52 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363487135 gn2sm15177636pbb.22 - gsmtp
[Sun Mar 17 02:24:52 2013] [error] data: (250, '2.0.0 OK 1363487135 gn2sm15177636pbb.22 - gsmtp')
[Sun Mar 17 02:24:52 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 02:24:52 2013] [error] reply: '221 2.0.0 closing connection gn2sm15177636pbb.22 - gsmtp\\r\\n'
[Sun Mar 17 02:24:52 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection gn2sm15177636pbb.22 - gsmtp
[Sun Mar 17 02:33:49 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 02:33:49 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 02:33:49 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 02:33:49 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 02:33:49 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 02:33:49 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 02:33:49 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 02:33:49 2013] [error] SIZE 35882577
[Sun Mar 17 02:33:49 2013] [error] 8BITMIME
[Sun Mar 17 02:33:49 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 02:33:49 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 02:33:49 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 02:33:49 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 02:33:49 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 02:33:49 2013] [error] send: u'mail FROM:<mbongos@gpails.com> size=1160\\r\\n'
[Sun Mar 17 02:33:49 2013] [error] reply: '250 2.1.0 OK pg7sm15212598pbc.5 - gsmtp\\r\\n'
[Sun Mar 17 02:33:49 2013] [error] reply: retcode (250); Msg: 2.1.0 OK pg7sm15212598pbc.5 - gsmtp
[Sun Mar 17 02:33:49 2013] [error] send: u'rcpt TO:<bbaggins@gpails.com>\\r\\n'
[Sun Mar 17 02:33:50 2013] [error] reply: '250 2.1.5 OK pg7sm15212598pbc.5 - gsmtp\\r\\n'
[Sun Mar 17 02:33:50 2013] [error] reply: retcode (250); Msg: 2.1.5 OK pg7sm15212598pbc.5 - gsmtp
[Sun Mar 17 02:33:50 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 02:33:50 2013] [error] reply: '354  Go ahead pg7sm15212598pbc.5 - gsmtp\\r\\n'
[Sun Mar 17 02:33:50 2013] [error] reply: retcode (354); Msg: Go ahead pg7sm15212598pbc.5 - gsmtp
[Sun Mar 17 02:33:50 2013] [error] data: (354, 'Go ahead pg7sm15212598pbc.5 - gsmtp')
[Sun Mar 17 02:33:50 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============5498509374657761555=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Mongo to a texting contest!\\r\\nFrom: MixedMessage challenges <mbongos@gpails.com>\\r\\nTo: bbaggins@gpails.com\\r\\nDate: Sun, 17 Mar 2013 02:33:49 +0000\\r\\nMessage-ID: <20130317023349.21364.76198@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============5498509374657761555==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============3167194240966523108=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============3167194240966523108==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============3167194240966523108==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEJpbGJvIEJhZ2dpbnMsPC9wPgo8cD5Zb3UgaGF2ZSBiZWVuIGNoYWxsZW5nZWQgYnkg\\r\\neW91ciBmcmllbmQgTW9uZ28gaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0\\r\\nZXIhPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwv\\r\\ncD4KPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============3167194240966523108==--\\r\\n--===============5498509374657761555==--\\r\\n.\\r\\n'
[Sun Mar 17 02:33:50 2013] [error] reply: '250 2.0.0 OK 1363487674 pg7sm15212598pbc.5 - gsmtp\\r\\n'
[Sun Mar 17 02:33:50 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363487674 pg7sm15212598pbc.5 - gsmtp
[Sun Mar 17 02:33:50 2013] [error] data: (250, '2.0.0 OK 1363487674 pg7sm15212598pbc.5 - gsmtp')
[Sun Mar 17 02:33:50 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 02:33:51 2013] [error] reply: '221 2.0.0 closing connection pg7sm15212598pbc.5 - gsmtp\\r\\n'
[Sun Mar 17 02:33:51 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection pg7sm15212598pbc.5 - gsmtp
[Sun Mar 17 02:35:20 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 02:35:20 2013] [error] SIZE 35882577
[Sun Mar 17 02:35:20 2013] [error] 8BITMIME
[Sun Mar 17 02:35:20 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 02:35:20 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 02:35:20 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 02:35:20 2013] [error] send: u'mail FROM:<mbongos@gpails.com> size=1143\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: '250 2.1.0 OK tm1sm15211148pbc.11 - gsmtp\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: retcode (250); Msg: 2.1.0 OK tm1sm15211148pbc.11 - gsmtp
[Sun Mar 17 02:35:20 2013] [error] send: u'rcpt TO:<p@face.com>\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: '250 2.1.5 OK tm1sm15211148pbc.11 - gsmtp\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: retcode (250); Msg: 2.1.5 OK tm1sm15211148pbc.11 - gsmtp
[Sun Mar 17 02:35:20 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: '354  Go ahead tm1sm15211148pbc.11 - gsmtp\\r\\n'
[Sun Mar 17 02:35:20 2013] [error] reply: retcode (354); Msg: Go ahead tm1sm15211148pbc.11 - gsmtp
[Sun Mar 17 02:35:20 2013] [error] data: (354, 'Go ahead tm1sm15211148pbc.11 - gsmtp')
[Sun Mar 17 02:35:20 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============8980115676934997068=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Mongo to a texting contest!\\r\\nFrom: MixedMessage challenges <mbongos@gpails.com>\\r\\nTo: p@face.com\\r\\nDate: Sun, 17 Mar 2013 02:35:20 +0000\\r\\nMessage-ID: <20130317023520.21364.56037@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============8980115676934997068==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============6598914029349903930=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============6598914029349903930==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============6598914029349903930==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IFBvb3AgRmFjZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBNb25nbyBpbiBhIHRleHQgb2ZmISE8L3A+CjxoMj5Zb3UgY2FuIGRvIGJldHRlciE8\\r\\nL2E+PC9oMj4KPHA+RG93bmxvYWQgdGhlIGFwcCBhbmQgYmVhdCB0aGVtIGhvcnJpYmx5PC9wPgo8\\r\\ncD4KQ2hlZXJzLAo8YnI+CkFsZXgKPGJyPgpNaXhlZE1lc3NhZ2UgdGVhbQo8L3A+\\r\\n\\r\\n--===============6598914029349903930==--\\r\\n--===============8980115676934997068==--\\r\\n.\\r\\n'
[Sun Mar 17 02:35:21 2013] [error] reply: '250 2.0.0 OK 1363487764 tm1sm15211148pbc.11 - gsmtp\\r\\n'
[Sun Mar 17 02:35:21 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363487764 tm1sm15211148pbc.11 - gsmtp
[Sun Mar 17 02:35:21 2013] [error] data: (250, '2.0.0 OK 1363487764 tm1sm15211148pbc.11 - gsmtp')
[Sun Mar 17 02:35:21 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 02:35:21 2013] [error] reply: '221 2.0.0 closing connection tm1sm15211148pbc.11 - gsmtp\\r\\n'
[Sun Mar 17 02:35:21 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection tm1sm15211148pbc.11 - gsmtp
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110] mod_wsgi (pid=17923): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110] Traceback (most recent call last):
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     return self.wsgi_app(environ, start_response)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     response = self.make_response(self.handle_exception(e))
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     response = self.full_dispatch_request()
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     rv = self.handle_user_exception(e)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     rv = self.dispatch_request()
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     return self.view_functions[rule.endpoint](**req.view_args)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/views.py", line 16, in invite_user_view
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     return user_definitions.invite()
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/user_definitions.py", line 136, in invite
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     create_user(new_name, new_email, "", False)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/user_definitions.py", line 54, in create_user
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     db.session.commit()
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/scoping.py", line 114, in do
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     return getattr(self.registry(), name)(*args, **kwargs)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 703, in commit
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     self.transaction.commit()
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 361, in commit
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     self._prepare_impl()
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 340, in _prepare_impl
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     self.session.flush()
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 1718, in flush
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     self._flush(objects)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/session.py", line 1789, in _flush
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     flush_context.execute()
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/unitofwork.py", line 331, in execute
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     rec.execute(self)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/unitofwork.py", line 475, in execute
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     uow
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/persistence.py", line 59, in save_obj
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     mapper, table, update)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/persistence.py", line 485, in _emit_update_statements
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     execute(statement, params)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1449, in execute
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     params)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1584, in _execute_clauseelement
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     compiled_sql, distilled_params
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1698, in _execute_context
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     context)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1691, in _execute_context
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     context)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/default.py", line 331, in do_execute
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]     cursor.execute(statement, parameters)
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110] ProgrammingError: (ProgrammingError) column "subscribed" is of type timestamp without time zone but expression is of type boolean
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110] LINE 1: UPDATE invite SET subscribed=false WHERE invite.id = 9
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]                                      ^
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110] HINT:  You will need to rewrite or cast the expression.
[Sun Mar 17 06:55:13 2013] [error] [client 171.66.130.110]  'UPDATE invite SET subscribed=%(subscribed)s WHERE invite.id = %(invite_id)s' {'invite_id': 9, 'subscribed': False}
[Sun Mar 17 08:23:09 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 08:23:09 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 08:23:09 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 08:23:09 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 08:23:09 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 08:23:09 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 08:23:09 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 08:23:09 2013] [error] SIZE 35882577
[Sun Mar 17 08:23:09 2013] [error] 8BITMIME
[Sun Mar 17 08:23:09 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 08:23:09 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 08:23:09 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 08:23:10 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 08:23:10 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 08:23:10 2013] [error] send: u'mail FROM:<alexjaustin@gmail.com> size=1151\\r\\n'
[Sun Mar 17 08:23:10 2013] [error] reply: '250 2.1.0 OK ol7sm16054092pbb.14 - gsmtp\\r\\n'
[Sun Mar 17 08:23:10 2013] [error] reply: retcode (250); Msg: 2.1.0 OK ol7sm16054092pbb.14 - gsmtp
[Sun Mar 17 08:23:10 2013] [error] send: u'rcpt TO:<gbluth@gpails.com>\\r\\n'
[Sun Mar 17 08:23:10 2013] [error] reply: '250 2.1.5 OK ol7sm16054092pbb.14 - gsmtp\\r\\n'
[Sun Mar 17 08:23:10 2013] [error] reply: retcode (250); Msg: 2.1.5 OK ol7sm16054092pbb.14 - gsmtp
[Sun Mar 17 08:23:10 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 08:23:10 2013] [error] reply: '354  Go ahead ol7sm16054092pbb.14 - gsmtp\\r\\n'
[Sun Mar 17 08:23:10 2013] [error] reply: retcode (354); Msg: Go ahead ol7sm16054092pbb.14 - gsmtp
[Sun Mar 17 08:23:10 2013] [error] data: (354, 'Go ahead ol7sm16054092pbb.14 - gsmtp')
[Sun Mar 17 08:23:10 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============8698678146767427021=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Alex to a texting contest!\\r\\nFrom: MixedMessage challenges <alexjaustin@gmail.com>\\r\\nTo: gbluth@gpails.com\\r\\nDate: Sun, 17 Mar 2013 08:23:10 +0000\\r\\nMessage-ID: <20130317082309.18423.2189@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============8698678146767427021==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============3157288151523419774=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============3157288151523419774==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============3157288151523419774==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEdvYiBCbHV0aCw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBBbGV4IGluIGEgdGV4dCBvZmYhITwvcD4KPGgyPllvdSBjYW4gZG8gYmV0dGVyITwv\\r\\nYT48L2gyPgo8cD5Eb3dubG9hZCB0aGUgYXBwIGFuZCBiZWF0IHRoZW0gaG9ycmlibHk8L3A+Cjxw\\r\\nPgpDaGVlcnMsCjxicj4KQWxleAo8YnI+Ck1peGVkTWVzc2FnZSB0ZWFtCjwvcD4=\\r\\n\\r\\n--===============3157288151523419774==--\\r\\n--===============8698678146767427021==--\\r\\n.\\r\\n'
[Sun Mar 17 08:23:11 2013] [error] reply: '250 2.0.0 OK 1363508635 ol7sm16054092pbb.14 - gsmtp\\r\\n'
[Sun Mar 17 08:23:11 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363508635 ol7sm16054092pbb.14 - gsmtp
[Sun Mar 17 08:23:11 2013] [error] data: (250, '2.0.0 OK 1363508635 ol7sm16054092pbb.14 - gsmtp')
[Sun Mar 17 08:23:11 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 08:23:11 2013] [error] reply: '221 2.0.0 closing connection ol7sm16054092pbb.14 - gsmtp\\r\\n'
[Sun Mar 17 08:23:11 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection ol7sm16054092pbb.14 - gsmtp
[Sun Mar 17 17:30:32 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 17:30:32 2013] [error] SIZE 35882577
[Sun Mar 17 17:30:32 2013] [error] 8BITMIME
[Sun Mar 17 17:30:32 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 17:30:32 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 17:30:32 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 17:30:32 2013] [error] send: u'mail FROM:<Joanne.yuan@gmail.com> size=1179\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: '250 2.1.0 OK i10sm17273196pbd.1 - gsmtp\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: retcode (250); Msg: 2.1.0 OK i10sm17273196pbd.1 - gsmtp
[Sun Mar 17 17:30:32 2013] [error] send: u'rcpt TO:<alexjaustin@gmail.com>\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: '250 2.1.5 OK i10sm17273196pbd.1 - gsmtp\\r\\n'
[Sun Mar 17 17:30:32 2013] [error] reply: retcode (250); Msg: 2.1.5 OK i10sm17273196pbd.1 - gsmtp
[Sun Mar 17 17:30:32 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 17:30:33 2013] [error] reply: '354  Go ahead i10sm17273196pbd.1 - gsmtp\\r\\n'
[Sun Mar 17 17:30:33 2013] [error] reply: retcode (354); Msg: Go ahead i10sm17273196pbd.1 - gsmtp
[Sun Mar 17 17:30:33 2013] [error] data: (354, 'Go ahead i10sm17273196pbd.1 - gsmtp')
[Sun Mar 17 17:30:33 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============6058297107420391150=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Joanne to a texting contest!\\r\\nFrom: MixedMessage challenges <Joanne.yuan@gmail.com>\\r\\nTo: alexjaustin@gmail.com\\r\\nDate: Sun, 17 Mar 2013 17:30:32 +0000\\r\\nMessage-ID: <20130317173031.18423.26906@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============6058297107420391150==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============5377371156198379028=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============5377371156198379028==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============5377371156198379028==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEFsZXhhbmRlciBBdXN0aW4g8J+SjCw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxl\\r\\nbmdlZCBieSB5b3VyIGZyaWVuZCBKb2FubmUgaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNh\\r\\nbiBkbyBiZXR0ZXIhPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBo\\r\\nb3JyaWJseTwvcD4KPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0K\\r\\nPC9wPg==\\r\\n\\r\\n--===============5377371156198379028==--\\r\\n--===============6058297107420391150==--\\r\\n.\\r\\n'
[Sun Mar 17 17:30:33 2013] [error] reply: '250 2.0.0 OK 1363541478 i10sm17273196pbd.1 - gsmtp\\r\\n'
[Sun Mar 17 17:30:33 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363541478 i10sm17273196pbd.1 - gsmtp
[Sun Mar 17 17:30:33 2013] [error] data: (250, '2.0.0 OK 1363541478 i10sm17273196pbd.1 - gsmtp')
[Sun Mar 17 17:30:33 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 17:30:33 2013] [error] reply: '221 2.0.0 closing connection i10sm17273196pbd.1 - gsmtp\\r\\n'
[Sun Mar 17 17:30:33 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection i10sm17273196pbd.1 - gsmtp
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44] mod_wsgi (pid=18423): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44] Traceback (most recent call last):
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]     return self.wsgi_app(environ, start_response)
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]     response = self.make_response(self.handle_exception(e))
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]     response = self.full_dispatch_request()
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]     rv = self.handle_user_exception(e)
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]     rv = self.dispatch_request()
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]     return self.view_functions[rule.endpoint](**req.view_args)
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]   File "/srv/ip/MixedMessage/app/views.py", line 24, in login_user_view
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]     return user_definitions.login(request.args['email'], request.args['password'])
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]   File "/srv/ip/MixedMessage/app/user_definitions.py", line 105, in login
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44]     user.append({
[Sun Mar 17 17:44:51 2013] [error] [client 171.66.133.44] AttributeError: 'User' object has no attribute 'append'
[Sun Mar 17 18:58:36 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 18:58:36 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 18:58:36 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 18:58:36 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 18:58:36 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 18:58:36 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 18:58:36 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 18:58:36 2013] [error] SIZE 35882577
[Sun Mar 17 18:58:36 2013] [error] 8BITMIME
[Sun Mar 17 18:58:36 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 18:58:36 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 18:58:36 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 18:58:37 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 18:58:37 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 18:58:37 2013] [error] send: u'mail FROM:<mbongos@gpails.com> size=1143\\r\\n'
[Sun Mar 17 18:58:37 2013] [error] reply: '250 2.1.0 OK pv7sm8575426pbb.33 - gsmtp\\r\\n'
[Sun Mar 17 18:58:37 2013] [error] reply: retcode (250); Msg: 2.1.0 OK pv7sm8575426pbb.33 - gsmtp
[Sun Mar 17 18:58:37 2013] [error] send: u'rcpt TO:<p@face.com>\\r\\n'
[Sun Mar 17 18:58:37 2013] [error] reply: '250 2.1.5 OK pv7sm8575426pbb.33 - gsmtp\\r\\n'
[Sun Mar 17 18:58:37 2013] [error] reply: retcode (250); Msg: 2.1.5 OK pv7sm8575426pbb.33 - gsmtp
[Sun Mar 17 18:58:37 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 18:58:37 2013] [error] reply: '354  Go ahead pv7sm8575426pbb.33 - gsmtp\\r\\n'
[Sun Mar 17 18:58:37 2013] [error] reply: retcode (354); Msg: Go ahead pv7sm8575426pbb.33 - gsmtp
[Sun Mar 17 18:58:37 2013] [error] data: (354, 'Go ahead pv7sm8575426pbb.33 - gsmtp')
[Sun Mar 17 18:58:37 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============0662363305364931269=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Mongo to a texting contest!\\r\\nFrom: MixedMessage challenges <mbongos@gpails.com>\\r\\nTo: p@face.com\\r\\nDate: Sun, 17 Mar 2013 18:58:37 +0000\\r\\nMessage-ID: <20130317185836.30519.63654@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============0662363305364931269==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============3651961021481891143=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============3651961021481891143==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============3651961021481891143==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IFBvb3AgRmFjZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBNb25nbyBpbiBhIHRleHQgb2ZmISE8L3A+CjxoMj5Zb3UgY2FuIGRvIGJldHRlciE8\\r\\nL2E+PC9oMj4KPHA+RG93bmxvYWQgdGhlIGFwcCBhbmQgYmVhdCB0aGVtIGhvcnJpYmx5PC9wPgo8\\r\\ncD4KQ2hlZXJzLAo8YnI+CkFsZXgKPGJyPgpNaXhlZE1lc3NhZ2UgdGVhbQo8L3A+\\r\\n\\r\\n--===============3651961021481891143==--\\r\\n--===============0662363305364931269==--\\r\\n.\\r\\n'
[Sun Mar 17 18:58:38 2013] [error] reply: '250 2.0.0 OK 1363546763 pv7sm8575426pbb.33 - gsmtp\\r\\n'
[Sun Mar 17 18:58:38 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363546763 pv7sm8575426pbb.33 - gsmtp
[Sun Mar 17 18:58:38 2013] [error] data: (250, '2.0.0 OK 1363546763 pv7sm8575426pbb.33 - gsmtp')
[Sun Mar 17 18:58:38 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 18:58:38 2013] [error] reply: '221 2.0.0 closing connection pv7sm8575426pbb.33 - gsmtp\\r\\n'
[Sun Mar 17 18:58:38 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection pv7sm8575426pbb.33 - gsmtp
[Sun Mar 17 19:06:41 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 19:06:41 2013] [error] SIZE 35882577
[Sun Mar 17 19:06:41 2013] [error] 8BITMIME
[Sun Mar 17 19:06:41 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 19:06:41 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 19:06:41 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 19:06:41 2013] [error] send: u'mail FROM:<mbongos@ppails.com> size=1150\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: '250 2.1.0 OK ab1sm17438541pbd.37 - gsmtp\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: retcode (250); Msg: 2.1.0 OK ab1sm17438541pbd.37 - gsmtp
[Sun Mar 17 19:06:41 2013] [error] send: u'rcpt TO:<gbluth@gpails.com>\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: '250 2.1.5 OK ab1sm17438541pbd.37 - gsmtp\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: retcode (250); Msg: 2.1.5 OK ab1sm17438541pbd.37 - gsmtp
[Sun Mar 17 19:06:41 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: '354  Go ahead ab1sm17438541pbd.37 - gsmtp\\r\\n'
[Sun Mar 17 19:06:41 2013] [error] reply: retcode (354); Msg: Go ahead ab1sm17438541pbd.37 - gsmtp
[Sun Mar 17 19:06:41 2013] [error] data: (354, 'Go ahead ab1sm17438541pbd.37 - gsmtp')
[Sun Mar 17 19:06:41 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============6358960234680914934=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Mongo to a texting contest!\\r\\nFrom: MixedMessage challenges <mbongos@ppails.com>\\r\\nTo: gbluth@gpails.com\\r\\nDate: Sun, 17 Mar 2013 19:06:41 +0000\\r\\nMessage-ID: <20130317190640.30519.36016@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============6358960234680914934==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============5878090621272993543=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============5878090621272993543==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============5878090621272993543==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEdvYiBCbHV0aCw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBNb25nbyBpbiBhIHRleHQgb2ZmISE8L3A+CjxoMj5Zb3UgY2FuIGRvIGJldHRlciE8\\r\\nL2E+PC9oMj4KPHA+RG93bmxvYWQgdGhlIGFwcCBhbmQgYmVhdCB0aGVtIGhvcnJpYmx5PC9wPgo8\\r\\ncD4KQ2hlZXJzLAo8YnI+CkFsZXgKPGJyPgpNaXhlZE1lc3NhZ2UgdGVhbQo8L3A+\\r\\n\\r\\n--===============5878090621272993543==--\\r\\n--===============6358960234680914934==--\\r\\n.\\r\\n'
[Sun Mar 17 19:06:42 2013] [error] reply: '250 2.0.0 OK 1363547247 ab1sm17438541pbd.37 - gsmtp\\r\\n'
[Sun Mar 17 19:06:42 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363547247 ab1sm17438541pbd.37 - gsmtp
[Sun Mar 17 19:06:42 2013] [error] data: (250, '2.0.0 OK 1363547247 ab1sm17438541pbd.37 - gsmtp')
[Sun Mar 17 19:06:42 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 19:06:42 2013] [error] reply: '221 2.0.0 closing connection ab1sm17438541pbd.37 - gsmtp\\r\\n'
[Sun Mar 17 19:06:42 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection ab1sm17438541pbd.37 - gsmtp
[Sun Mar 17 19:07:45 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 19:07:46 2013] [error] SIZE 35882577
[Sun Mar 17 19:07:46 2013] [error] 8BITMIME
[Sun Mar 17 19:07:46 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 19:07:46 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 19:07:46 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 19:07:46 2013] [error] send: u'mail FROM:<mbongos@ppails.com> size=1160\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: '250 2.1.0 OK iv3sm17433341pbc.40 - gsmtp\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: retcode (250); Msg: 2.1.0 OK iv3sm17433341pbc.40 - gsmtp
[Sun Mar 17 19:07:46 2013] [error] send: u'rcpt TO:<bbaggins@gpails.com>\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: '250 2.1.5 OK iv3sm17433341pbc.40 - gsmtp\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: retcode (250); Msg: 2.1.5 OK iv3sm17433341pbc.40 - gsmtp
[Sun Mar 17 19:07:46 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: '354  Go ahead iv3sm17433341pbc.40 - gsmtp\\r\\n'
[Sun Mar 17 19:07:46 2013] [error] reply: retcode (354); Msg: Go ahead iv3sm17433341pbc.40 - gsmtp
[Sun Mar 17 19:07:46 2013] [error] data: (354, 'Go ahead iv3sm17433341pbc.40 - gsmtp')
[Sun Mar 17 19:07:46 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============4207615608178460294=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Mongo to a texting contest!\\r\\nFrom: MixedMessage challenges <mbongos@ppails.com>\\r\\nTo: bbaggins@gpails.com\\r\\nDate: Sun, 17 Mar 2013 19:07:46 +0000\\r\\nMessage-ID: <20130317190745.30519.69001@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============4207615608178460294==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============9036301063157437009=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============9036301063157437009==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============9036301063157437009==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEJpbGJvIEJhZ2dpbnMsPC9wPgo8cD5Zb3UgaGF2ZSBiZWVuIGNoYWxsZW5nZWQgYnkg\\r\\neW91ciBmcmllbmQgTW9uZ28gaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0\\r\\nZXIhPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwv\\r\\ncD4KPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============9036301063157437009==--\\r\\n--===============4207615608178460294==--\\r\\n.\\r\\n'
[Sun Mar 17 19:07:47 2013] [error] reply: '250 2.0.0 OK 1363547312 iv3sm17433341pbc.40 - gsmtp\\r\\n'
[Sun Mar 17 19:07:47 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363547312 iv3sm17433341pbc.40 - gsmtp
[Sun Mar 17 19:07:47 2013] [error] data: (250, '2.0.0 OK 1363547312 iv3sm17433341pbc.40 - gsmtp')
[Sun Mar 17 19:07:47 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 19:07:47 2013] [error] reply: '221 2.0.0 closing connection iv3sm17433341pbc.40 - gsmtp\\r\\n'
[Sun Mar 17 19:07:47 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection iv3sm17433341pbc.40 - gsmtp
[Sun Mar 17 19:08:51 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 19:08:51 2013] [error] SIZE 35882577
[Sun Mar 17 19:08:51 2013] [error] 8BITMIME
[Sun Mar 17 19:08:51 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 19:08:51 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 19:08:51 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 19:08:51 2013] [error] send: u'mail FROM:<mbongos@ppails.com> size=1150\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: '250 2.1.0 OK ve7sm5268170pab.11 - gsmtp\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: retcode (250); Msg: 2.1.0 OK ve7sm5268170pab.11 - gsmtp
[Sun Mar 17 19:08:51 2013] [error] send: u'rcpt TO:<joanne.yuan@gmail.com>\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: '250 2.1.5 OK ve7sm5268170pab.11 - gsmtp\\r\\n'
[Sun Mar 17 19:08:51 2013] [error] reply: retcode (250); Msg: 2.1.5 OK ve7sm5268170pab.11 - gsmtp
[Sun Mar 17 19:08:51 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 19:08:52 2013] [error] reply: '354  Go ahead ve7sm5268170pab.11 - gsmtp\\r\\n'
[Sun Mar 17 19:08:52 2013] [error] reply: retcode (354); Msg: Go ahead ve7sm5268170pab.11 - gsmtp
[Sun Mar 17 19:08:52 2013] [error] data: (354, 'Go ahead ve7sm5268170pab.11 - gsmtp')
[Sun Mar 17 19:08:52 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============7998371136958295402=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Mongo to a texting contest!\\r\\nFrom: MixedMessage challenges <mbongos@ppails.com>\\r\\nTo: joanne.yuan@gmail.com\\r\\nDate: Sun, 17 Mar 2013 19:08:51 +0000\\r\\nMessage-ID: <20130317190851.30519.78876@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============7998371136958295402==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============6047889513552667597=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============6047889513552667597==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============6047889513552667597==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEpvYW5uZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3VyIGZy\\r\\naWVuZCBNb25nbyBpbiBhIHRleHQgb2ZmISE8L3A+CjxoMj5Zb3UgY2FuIGRvIGJldHRlciE8L2E+\\r\\nPC9oMj4KPHA+RG93bmxvYWQgdGhlIGFwcCBhbmQgYmVhdCB0aGVtIGhvcnJpYmx5PC9wPgo8cD4K\\r\\nQ2hlZXJzLAo8YnI+CkFsZXgKPGJyPgpNaXhlZE1lc3NhZ2UgdGVhbQo8L3A+\\r\\n\\r\\n--===============6047889513552667597==--\\r\\n--===============7998371136958295402==--\\r\\n.\\r\\n'
[Sun Mar 17 19:08:52 2013] [error] reply: '250 2.0.0 OK 1363547377 ve7sm5268170pab.11 - gsmtp\\r\\n'
[Sun Mar 17 19:08:52 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363547377 ve7sm5268170pab.11 - gsmtp
[Sun Mar 17 19:08:52 2013] [error] data: (250, '2.0.0 OK 1363547377 ve7sm5268170pab.11 - gsmtp')
[Sun Mar 17 19:08:52 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 19:08:53 2013] [error] reply: '221 2.0.0 closing connection ve7sm5268170pab.11 - gsmtp\\r\\n'
[Sun Mar 17 19:08:53 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection ve7sm5268170pab.11 - gsmtp
[Sun Mar 17 19:23:15 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 19:23:15 2013] [error] SIZE 35882577
[Sun Mar 17 19:23:15 2013] [error] 8BITMIME
[Sun Mar 17 19:23:15 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 19:23:15 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 19:23:15 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 19:23:15 2013] [error] send: u'mail FROM:<mbongos@ppails.com> size=1143\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: '250 2.1.0 OK ti8sm17487396pbc.12 - gsmtp\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: retcode (250); Msg: 2.1.0 OK ti8sm17487396pbc.12 - gsmtp
[Sun Mar 17 19:23:15 2013] [error] send: u'rcpt TO:<p@face.com>\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: '250 2.1.5 OK ti8sm17487396pbc.12 - gsmtp\\r\\n'
[Sun Mar 17 19:23:15 2013] [error] reply: retcode (250); Msg: 2.1.5 OK ti8sm17487396pbc.12 - gsmtp
[Sun Mar 17 19:23:15 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 19:23:16 2013] [error] reply: '354  Go ahead ti8sm17487396pbc.12 - gsmtp\\r\\n'
[Sun Mar 17 19:23:16 2013] [error] reply: retcode (354); Msg: Go ahead ti8sm17487396pbc.12 - gsmtp
[Sun Mar 17 19:23:16 2013] [error] data: (354, 'Go ahead ti8sm17487396pbc.12 - gsmtp')
[Sun Mar 17 19:23:16 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============3957811693774700711=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Mongo to a texting contest!\\r\\nFrom: MixedMessage challenges <mbongos@ppails.com>\\r\\nTo: p@face.com\\r\\nDate: Sun, 17 Mar 2013 19:23:15 +0000\\r\\nMessage-ID: <20130317192315.30519.69813@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============3957811693774700711==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============6048139708838414487=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============6048139708838414487==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============6048139708838414487==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IFBvb3AgRmFjZSw8L3A+CjxwPllvdSBoYXZlIGJlZW4gY2hhbGxlbmdlZCBieSB5b3Vy\\r\\nIGZyaWVuZCBNb25nbyBpbiBhIHRleHQgb2ZmISE8L3A+CjxoMj5Zb3UgY2FuIGRvIGJldHRlciE8\\r\\nL2E+PC9oMj4KPHA+RG93bmxvYWQgdGhlIGFwcCBhbmQgYmVhdCB0aGVtIGhvcnJpYmx5PC9wPgo8\\r\\ncD4KQ2hlZXJzLAo8YnI+CkFsZXgKPGJyPgpNaXhlZE1lc3NhZ2UgdGVhbQo8L3A+\\r\\n\\r\\n--===============6048139708838414487==--\\r\\n--===============3957811693774700711==--\\r\\n.\\r\\n'
[Sun Mar 17 19:23:17 2013] [error] reply: '250 2.0.0 OK 1363548241 ti8sm17487396pbc.12 - gsmtp\\r\\n'
[Sun Mar 17 19:23:17 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363548241 ti8sm17487396pbc.12 - gsmtp
[Sun Mar 17 19:23:17 2013] [error] data: (250, '2.0.0 OK 1363548241 ti8sm17487396pbc.12 - gsmtp')
[Sun Mar 17 19:23:17 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 19:23:17 2013] [error] reply: '221 2.0.0 closing connection ti8sm17487396pbc.12 - gsmtp\\r\\n'
[Sun Mar 17 19:23:17 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection ti8sm17487396pbc.12 - gsmtp
[Sun Mar 17 19:25:56 2013] [error] send: 'ehlo ip-10-166-214-198.us-west-1.compute.internal\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: '250-mx.google.com at your service, [54.241.22.27]\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: '250-SIZE 35882577\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: '250-8BITMIME\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: '250-AUTH LOGIN PLAIN XOAUTH XOAUTH2\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: '250 ENHANCEDSTATUSCODES\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: retcode (250); Msg: mx.google.com at your service, [54.241.22.27]
[Sun Mar 17 19:25:57 2013] [error] SIZE 35882577
[Sun Mar 17 19:25:57 2013] [error] 8BITMIME
[Sun Mar 17 19:25:57 2013] [error] AUTH LOGIN PLAIN XOAUTH XOAUTH2
[Sun Mar 17 19:25:57 2013] [error] ENHANCEDSTATUSCODES
[Sun Mar 17 19:25:57 2013] [error] send: 'AUTH PLAIN AGFsZXhqYXVzdGluQGdtYWlsLmNvbQBtYWVwaDAwODk=\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: '235 2.7.0 Accepted\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: retcode (235); Msg: 2.7.0 Accepted
[Sun Mar 17 19:25:57 2013] [error] send: u'mail FROM:<mbongos@ppails.com> size=1160\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: '250 2.1.0 OK pv7sm8629253pbb.33 - gsmtp\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: retcode (250); Msg: 2.1.0 OK pv7sm8629253pbb.33 - gsmtp
[Sun Mar 17 19:25:57 2013] [error] send: u'rcpt TO:<bbaggins@gpails.com>\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: '250 2.1.5 OK pv7sm8629253pbb.33 - gsmtp\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: retcode (250); Msg: 2.1.5 OK pv7sm8629253pbb.33 - gsmtp
[Sun Mar 17 19:25:57 2013] [error] send: 'data\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: '354  Go ahead pv7sm8629253pbb.33 - gsmtp\\r\\n'
[Sun Mar 17 19:25:57 2013] [error] reply: retcode (354); Msg: Go ahead pv7sm8629253pbb.33 - gsmtp
[Sun Mar 17 19:25:57 2013] [error] data: (354, 'Go ahead pv7sm8629253pbb.33 - gsmtp')
[Sun Mar 17 19:25:57 2013] [error] send: 'Content-Type: multipart/mixed; boundary="===============1686220869546884389=="\\r\\nMIME-Version: 1.0\\r\\nSubject: You have been challenged by Mongo to a texting contest!\\r\\nFrom: MixedMessage challenges <mbongos@ppails.com>\\r\\nTo: bbaggins@gpails.com\\r\\nDate: Sun, 17 Mar 2013 19:25:57 +0000\\r\\nMessage-ID: <20130317192556.30519.58180@ip-10-166-214-198.us-west-1.compute.internal>\\r\\n\\r\\n--===============1686220869546884389==\\r\\nContent-Type: multipart/alternative;\\r\\n boundary="===============5935863814136145157=="\\r\\nMIME-Version: 1.0\\r\\n\\r\\n--===============5935863814136145157==\\r\\nContent-Type: text/plain; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\n\\r\\n--===============5935863814136145157==\\r\\nContent-Type: text/html; charset="utf-8"\\r\\nMIME-Version: 1.0\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPHA+SGV5IEJpbGJvIEJhZ2dpbnMsPC9wPgo8cD5Zb3UgaGF2ZSBiZWVuIGNoYWxsZW5nZWQgYnkg\\r\\neW91ciBmcmllbmQgTW9uZ28gaW4gYSB0ZXh0IG9mZiEhPC9wPgo8aDI+WW91IGNhbiBkbyBiZXR0\\r\\nZXIhPC9hPjwvaDI+CjxwPkRvd25sb2FkIHRoZSBhcHAgYW5kIGJlYXQgdGhlbSBob3JyaWJseTwv\\r\\ncD4KPHA+CkNoZWVycywKPGJyPgpBbGV4Cjxicj4KTWl4ZWRNZXNzYWdlIHRlYW0KPC9wPg==\\r\\n\\r\\n--===============5935863814136145157==--\\r\\n--===============1686220869546884389==--\\r\\n.\\r\\n'
[Sun Mar 17 19:25:58 2013] [error] reply: '250 2.0.0 OK 1363548403 pv7sm8629253pbb.33 - gsmtp\\r\\n'
[Sun Mar 17 19:25:58 2013] [error] reply: retcode (250); Msg: 2.0.0 OK 1363548403 pv7sm8629253pbb.33 - gsmtp
[Sun Mar 17 19:25:58 2013] [error] data: (250, '2.0.0 OK 1363548403 pv7sm8629253pbb.33 - gsmtp')
[Sun Mar 17 19:25:58 2013] [error] send: 'quit\\r\\n'
[Sun Mar 17 19:25:58 2013] [error] reply: '221 2.0.0 closing connection pv7sm8629253pbb.33 - gsmtp\\r\\n'
[Sun Mar 17 19:25:58 2013] [error] reply: retcode (221); Msg: 2.0.0 closing connection pv7sm8629253pbb.33 - gsmtp
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110] mod_wsgi (pid=2936): Exception occurred processing WSGI script '/srv/ip/MixedMessage/mixedmessage.wsgi'.
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110] Traceback (most recent call last):
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1701, in __call__
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]     return self.wsgi_app(environ, start_response)
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1689, in wsgi_app
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]     response = self.make_response(self.handle_exception(e))
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1687, in wsgi_app
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]     response = self.full_dispatch_request()
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1360, in full_dispatch_request
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]     rv = self.handle_user_exception(e)
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1358, in full_dispatch_request
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]     rv = self.dispatch_request()
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1344, in dispatch_request
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]     return self.view_functions[rule.endpoint](**req.view_args)
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/views.py", line 12, in create_user_view
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]     return user_definitions.create()
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]   File "/srv/ip/MixedMessage/app/user_definitions.py", line 18, in create
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110]     'name' : user.name,
[Mon Mar 18 00:19:03 2013] [error] [client 171.66.130.110] AttributeError: 'unicode' object has no attribute 'name'
